content/en/docs/setup/platform-setup/docker/index.md

```diff
@@ -12,6 +12,13 @@ owner: istio/wg-environments-maintainers
 test: no
 ---
 
+{{< warning >}}
```

Comment:
Noted that I believe this has been tested on nearly all releases.

Reply:
Yes, I went back and found that you did this. Thanks for fact checking. Resolved.